packages/components/modules/notifications/native/NotificationBellWithBadge/styles.ts

```diff
@@ -0,0 +1,17 @@
+import { Theme } from '@baseapp-frontend/design-system/styles/native'
+
+import { StyleSheet } from 'react-native'
+
+export const createStyles = (theme: Theme) =>
+  StyleSheet.create({
+    badge: {
+      position: 'absolute',
+      top: -10,
+      right: -10,
+      backgroundColor: theme.colors.error.main,
+      color: theme.colors.error.contrast,
+      ...theme.typography.caption,
```

Comment:
this isomething we might want to do inside the Badge component, sow e do it only once. Maybe we could wrap a Text over the children like
`  <BadgeComponent {...props}>
    <Text variant='caption'>{children}</Text>
  </BadgeComponent>
 `
 but then the children must be a string 🤔